common/browser.go

```diff
@@ -377,10 +378,7 @@ func (b *Browser) onDetachedFromTarget(ev *target.EventDetachedFromTarget) {
 }
 
 func (b *Browser) newPageInContext(id cdp.BrowserContextID) (*Page, error) {
-	b.contextsMu.RLock()
-	browserCtx, ok := b.contexts[id]
-	b.contextsMu.RUnlock()
-	if !ok {
+	if b.context == nil || b.context.id != id {
 		return nil, fmt.Errorf("missing browser context: %s", id)
```

Comment:
Do you think we should make this in line with the other error message? https://github.com/grafana/xk6-browser/pull/929/files#r1228001140

Reply:
Thanks!